hare/algorithm.go

```diff
@@ -598,7 +603,19 @@ func (proc *ConsensusProcess) endOfRound3() {
 	proc.notifySent = true
 }
 
-func (proc *ConsensusProcess) isEligible() bool {
+func (proc *ConsensusProcess) shouldParticipate() bool {
+	// query if identity is active
+	res, _, err := proc.stateQuerier.IsIdentityActive(proc.signing.PublicKey().String(), types.LayerID(proc.instanceId))
+	if err != nil {
+		proc.Error("Error checking for identity err=%v", err)
```

Comment:
Should be "error checking node's activity". Also, add layer id (every log print from now should include the layer id)

Reply:
I changed to "Error checking our identity for activeness"
Don't want to insert ' because of the JSON